### PR TITLE
improve styling of download toast

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

* Optimizes video render mode to record individual MP4 files for each session event chunk and combine them with ffmpeg
* Updates the session exports panel in project settings to show the mp4 files correctly.
* Records rendering errors to track exports that have failed.

## How did you test this change?

Screenshot 2023-09-18 at 3 53 23 PM

exporting large session from our project.
https://highlight-session-render.s3.us-east-2.amazonaws.com/1/In4fdHUUnHDHbiwOhsEAfIlym8Os.mp4

exporting large sessions from project 5403 - no longer OOMs and exports are smooth

Screenshot 2023-09-18 at 3 42 33 PM

## Are there any deployment considerations?

No

## Does this work require review from our design team?

Working with @julian-highlight on the styling of the exports page.

### CODE REVIEW

**Comment by Vadman97** [comment]
@julian-highlight this should be ready with your latest changes
Screenshot 2023-09-19 at 11 32 21 AM

File: backend/lambda-functions/sessionExport/main.go [hunk 1]
```diff
@@ -25,9 +25,10 @@ func main() {
 
 	h := handlers.NewHandlers()
 	input := utils.SaveSessionExportInput{
-		SessionID:    311983171,
+		SessionID:    30000031,
 		Type:         model.SessionExportFormatMP4,
 		URL:          "https://highlight-session-render.s3.us-east-2.amazonaws.com/1/311983171.mp4",
```
Comment (SpennyNDaJets) [inline backend/lambda-functions/sessionExport/main.go:30]: Not sure if this is related to the failing spec, but should the session id be updated here?

**Review by SpennyNDaJets — APPROVED** [review]
Click test looks good to me!

A couple comments:
1) not sure if files can be downloaded locally or may have something off, but seems to be in an indefinite "in-progress" state when I tried
2) The toast link takes to the correct page, but was a little confused on where to look at first because the "Session Export Requests" was beneath the fold

File: backend/lambda-functions/sessionExport/main.go (hunk 1 above)
Comment (Vadman97) [reply backend/lambda-functions/sessionExport/main.go:30]: the test needs access to postgres now - fixed!

**Comment by Vadman97** [comment]
thanks @SpennyNDaJets 

* not sure if files can be downloaded locally or may have something off, but seems to be in an indefinite "in-progress" state when I tried

this doesn't work locally because we aren't sending a `dev` prefix to find your sessions in s3. not super straightforward to make it compatible with the `/dev` storage prefix, so it's only going to work in prod for now

* The toast link takes to the correct page, but was a little confused on where to look at first because the "Session Export Requests" was beneath the fold

great call out! making the link point to the exports session

**Review by julian-highlight — COMMENTED** [review]
Click-test looks great to me

**Pull request merged**